<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Principal">
   <widget class="QWidget" name="Identification" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>691</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 132, 255);</string>
    </property>
    <widget class="QLineEdit" name="linePrenom">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelNom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonValider">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>120</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNom">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelPrenom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAlert">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Veuillez renseigner vos identifiants.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetPurse" native="true">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>219</y>
      <width>691</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 132, 255);</string>
    </property>
    <widget class="QLabel" name="labelTitre">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Importation du purse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonParcourir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Parcourir</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textColler">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>631</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonCollerValider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTriown">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TRIOwn</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelResultat">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Résultat</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonQuitter">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>670</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 132, 255);</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
